### PR TITLE
Fix DesignTools.createMissingStaticSitePins() when only LUT6

### PR DESCRIPTION
**Pull request by eddieh-xlnx**
... to not connect A6 to VCC unless there is a corresponding [A-H]5LUT cell placed.

### CODE REVIEW

File: src/com/xilinx/rapidwright/design/DesignTools.java [hunk 1]
```diff
@@ -3029,6 +3029,11 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
                     net = si.getDesign().getGndNet();
                 }
             }
+            String belName = belPin.getBELName();
+            if (DesignTools.isBELALut(belName) && si.getCell(belName.replace('6', '5')) == null) {
```
Comment (clavin-xlnx) [inline src/com/xilinx/rapidwright/design/DesignTools.java:3033]: Should we use `belPin.getBEL().isLUT()` over `DesignTools.isBELALut()`?

**Review by clavin-xlnx — APPROVED** [review]

File: src/com/xilinx/rapidwright/design/DesignTools.java (hunk 1 above)
Comment (eddieh-xlnx) [reply src/com/xilinx/rapidwright/design/DesignTools.java:3033]: Should we consider deprecating the latter?

Comment (eddieh-xlnx) [inline src/com/xilinx/rapidwright/design/DesignTools.java:3033]:
```suggestion
            String belName = belPin.getBELName();
            if (bel.isLUT() && si.getCell(belName.replace('6', '5')) == null) {
```

Comment (clavin-xlnx) [reply src/com/xilinx/rapidwright/design/DesignTools.java]: I don't think the `bel` variable exists.

Comment (eddieh-xlnx) [reply src/com/xilinx/rapidwright/design/DesignTools.java]: Let's leave it then.

File: src/com/xilinx/rapidwright/design/DesignTools.java [hunk 2]
```diff
@@ -3029,6 +3029,11 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
                     net = si.getDesign().getGndNet();
                 }
             }
+            String belName = belPin.getBELName();
+            if (bel.isLUT() && si.getCell(belName.replace('6', '5')) == null) {
```
Comment (eddieh-xlnx) [inline src/com/xilinx/rapidwright/design/DesignTools.java:3033]:
```suggestion
            if (belPin.getBEL().isLUT() && si.getCell(belName.replace('6', '5')) == null) {
```

**Pull request merged**